### ISSUE TITLE
[BUG] Food Degredation Prevention not working for Eitr

### ISSUE THREAD

**Issue by Crow-x** [valheimPlus/ValheimPlus#743]
Windows Client Version 0.9.9.11

**Describe the bug**
With disableFoodDegradation = true, the new Eitr stat (mana) still will degrade over time when using eitr based food.

**Comment by increddibelly** [comment]
that would be a great idea for a new feature, and I see how you think it's a bug <3 
I'll go have a look.

**Comment by Crow-x** [comment]
Yeah, I debated if it should be new feature or bug. Sorry if I choose wrong! Thanks :)

**Comment by Kherae** [comment]
hate the game's food degradation system as a whole. without the setting affecting eiter I can barely get off any spells in the later 'half'. hope this gets addressed soon :/

**Comment by healiha** [comment]
I'll take a look at this, I haven't had time to play Valheim since the patch release so I didn't try the new Eitr stat food(s).

Will keep you updated once it's done.

**Comment by Grantapher** [comment]
#769 looks like it fixes this.
